### PR TITLE
184: Archive 2020 website

### PR DESCRIPTION
**Pull request by hotwebmatter**
This PR attempts to resolve issue #184:
https://github.com/Drupal-NYC/drupalcampnyc/issues/184

I used the Tome module to generate a static HTML site based on last year's website.

(See https://tome.fyi/docs/installing-existing-site/ for details.)

This will need some testing and debugging. The basic export is happening, but it does not look correct in the browser when I view it using `file://` and the path to the `html` directory.

### CODE REVIEW

**Comment by hotwebmatter** [comment]
@jdleonard Actually I am going to close this PR since I do not believe this branch should be merged with `master`.

File: web/sites/default/settings.php [hunk 1]
```diff
@@ -175,3 +175,13 @@
...
+
+/**
+ * Tome export settings.
+ *
+ * See https://tome.fyi/docs/installing-existing-site/ for details.
+ */
+$settings['tome_files_directory'] = '../files';
+$settings['tome_content_directory'] = '../content';
+$settings['tome_static_directory'] = '../html';
+$config_directories['sync'] = '../config';
```
Comment (jdleonard) [inline web/sites/default/settings.php:187]: This last line is [deprecated](https://www.drupal.org/node/3018145).

File: config-development/devel.settings.yml [hunk 2]
```diff
@@ -1,10 +0,0 @@
-page_alter: false
```
Comment (jdleonard) [inline config-development/devel.settings.yml:1]: This and some other files look like they were removed by accident.

**Review by jdleonard — COMMENTED** [review]
@hotwebmatter I haven't yet had a chance to review locally, but here's a quick code review.

**Pull request closed without merging**